README.md

```diff
@@ -1,45 +1 @@
-<img align="right" alt="Xata Examples logo: a test tube and a Xatafly" src="/docs/logo.png" width="200" />
-
-# Xata Examples
-
-[![Open in Codeflow](https://developer.stackblitz.com/img/open_in_codeflow.svg)](https:///pr.new/xataio/examples)
-
-## Apps 🏗
-
-- **getting-started**
-
-  - Prefix: `gettingstarted-`
-  - Framework
-  - Minimal Dependencies
-  - Show the results of following the Xata getting started guides
-
-- **starter**
-
-  - Prefix: `starter-`
-  - Framework / Architecture Integrations
-  - Minimal Dependencies
-  - Ready to Use
-
-- **samples**
-  - Prefix: `sample-`
-  - Real-World Integrations
-  - Production-Grade Solutions
-  - Reverse Engineering / Strategy Insights
-
-## Get and Offer Help ❓
-
-We value all kinds of contributions. Please check our [CONTRIBUTING](/CONTRIBUTING.md).
-
-Also, remember you're always invited to join us at [Xata Serverless DB Community on Discord](https://xata.io/discord).
-
-## Contributors
-
-<a href="https://github.com/xataio/examples/graphs/contributors">
-  <img src="https://contrib.rocks/image?repo=xataio/examples" />
-</a>
-
-Want to see yourself up there? [Contribute](https://github.com/xataio/examples/blob/a73693f6a0e07565a209544498de1da5030dde13/CONTRIBUTING.md).
-
----
-
-<img alt="Xatafly, Xata's logo" src="https://raw.githubusercontent.com/xataio/vscode-extension/2e3d0b877cf6aff1e0fc717e05ada714465ca783/doc/xata-icon-128.png" width="24" />
+This repository houses various examples linked to from our [documentation](https://xata.io/docs). PRs are welcome.
```

Comment:
If we still welcome PRs, shouldn't we keep the `CONTRIBUTING.md` and still link that here?

Reply:
This change removes the CTA to join the Discord as well. 

Alex F. mentioned at the offsite that we want to increase avenues for engagement with the community and discovery of the Discord. I don't know what the click-through rate is from here, so this may not have been a well-functioning CTA, but wanted to call it out just in case.